app/views/devise/unlocks/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend unlock instructions</h2>
```

Comment:
What is this workflow and is it needed?

Reply:
It's a built in devise workflow, would only be used if :lockable and :unlockable was added to the user model. Would be used if we wanted to lock accounts after a certain number of failed attempts. Probably not useful for us though.